<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UF_AddUser</class>
 <widget class="QMainWindow" name="UF_AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New User</string>
  </property>
  <property name="styleSheet">
   <string notr="true">BACKGROUND-COLOR: rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="Widget_AddUser">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_LAB_EmployeeID">
     <item row="0" column="0">
      <widget class="QLabel" name="LAB_EmployeeID">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Employee ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_LAB_UserLogin">
     <item row="0" column="0">
      <widget class="QLabel" name="LAB_UserLogin">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;User Login:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_LAB_Password">
     <item row="0" column="0">
      <widget class="QLabel" name="LAB_Password">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_LAB_FirstName">
     <item row="0" column="0">
      <widget class="QLabel" name="LAB_FirstName">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;First Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_LAB_LastName">
     <item row="0" column="0">
      <widget class="QLabel" name="LAB_LastName">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Last Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_Lab_Email">
     <item row="0" column="0">
      <widget class="QLabel" name="LAB_Email">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;E-Mail:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_LAB_Admin">
     <item row="0" column="0">
      <widget class="QLabel" name="LAB_Admin">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Admin Priveleges:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_TB_EmployeeID">
     <item row="0" column="0">
      <widget class="QLineEdit" name="TB_EmployeeID">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(245, 245, 245);
BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_TB_UserLogin">
     <item row="0" column="0">
      <widget class="QLineEdit" name="TB_UserLogin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(245, 245, 245);
BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_TB_Password">
     <item row="0" column="0">
      <widget class="QLineEdit" name="TB_Password">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(245, 245, 245);
BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_TB_FirstName">
     <item row="0" column="0">
      <widget class="QLineEdit" name="TB_FirstName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(245, 245, 245);
BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_TB_LastName">
     <item row="0" column="0">
      <widget class="QLineEdit" name="TB_LastName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(245, 245, 245);
BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_TB_Email">
     <item row="0" column="0">
      <widget class="QLineEdit" name="TB_Email">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(245, 245, 245);
BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_14">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_CB_Admin">
     <item row="0" column="0">
      <widget class="QCheckBox" name="CB_Admin">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">BORDER-COLOR: rgb(245, 245, 245);
BORDER-RADIUS: 7px;
BORDER-WIDTH: 1px;
BORDER-STYLE: outset;</string>
       </property>
       <property name="text">
        <string>Give User Administrative Permissions</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_15">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_LAB_AddUser">
     <item row="0" column="0">
      <widget class="QLabel" name="LAB_AddUser">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(208, 248, 255);
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Add New User&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_PB_Cancel">
     <item row="0" column="0">
      <widget class="QPushButton" name="PB_Cancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change your password, create new users, and delete existing users&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(255, 175, 177);
BORDER-COLOR: rgb(255, 175. 177);
BORDER-RADIUS: 10px;
BORDER-STYLE: outset;
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_17">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GL_PB_Confirm">
     <item row="0" column="0">
      <widget class="QPushButton" name="PB_Confirm">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm username and password to log in&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">BACKGROUND-COLOR: rgb(170, 255, 127);
BORDER-COLOR: rgb(170, 255, 127);
BORDER-RADIUS: 10px;
BORDER-STYLE: outset;
</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
